sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -3094,8 +3095,16 @@ class BlobAPITest extends APISpec {
     }
 
     def "Get Blob Name"() {
+        setup:
+        bc = cc.getBlobClient(name)
+
         expect:
-        blobName == bc.getBlobName()
+        resultName == bc.getBlobName()
+
+        where:
+        name                                | resultName
```

Comment:
how about `expected name?`

Reply:
`inputName` and `outputName`?

Reply:
```
def inputName = "bla"
def expectedOutputName = "bla I think I'll get"
def outputName = bc.getBlobName()

assert outputName = expectedOutputName
```
That's if I were most explicit.

`inputName` and `expectedOutputName` might suffice.